src/mds/mdstypes.cc

```diff
@@ -551,25 +551,29 @@ void old_inode_t::generate_test_instances(list<old_inode_t*>& ls)
  */
 void fnode_t::encode(bufferlist &bl) const
 {
-  ENCODE_START(2, 2, bl);
+  ENCODE_START(3, 2, bl);
```

Comment:
I don't think we want to let the damage_flags be dropped; we should bump up the required version as well.